### PR TITLE
PD-3883: Merge cluster support

### PR DESCRIPTION
**Pull request by saxaathlete**

### CODE REVIEW

**Review by cthulhu — APPROVED** [review]

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -5,13 +5,4 @@
...
-
-RSpec.configure do |config|
-  config.before(:each) do
-    Seoshop.configure do |conf|
-      conf.app_key = 'app_key'
-      conf.secret = 'secret'
-    end
-  end
-end
+require './lib/seoshop-api'
```
Comment (hyxnat) [inline spec/spec_helper.rb]: press enter here please

File: lib/seoshop-api/client.rb [hunk 2]
```diff
@@ -33,21 +33,43 @@ class Client
...
       @access_token = access_token
       @shop_language = shop_language
-      @url = url
+      @url = get_url(cluster_id)
       @parallel_requests = parallel_requests
     end
 
+    def get_url(cluster_id)
+      case cluster_id
+        when 'eu1'
```
Comment (hyxnat) [inline lib/seoshop-api/client.rb]: if else?

File: lib/seoshop-api/client.rb [hunk 3]
```diff
@@ -33,21 +33,43 @@ class Client
     include Seoshop::Catalog
     include Seoshop::FetchResourceHelper
 
+    attr_accessor :api_key
+    attr_accessor :api_secret
     attr_accessor :access_token
     attr_accessor :shop_language
 
+    SERVER_EU1_LIVE  = 'https://api.webshopapp.com/'
```
Comment (hyxnat) [inline lib/seoshop-api/client.rb]: is it possible to make this part of the config

Comment (hyxnat) [inline lib/seoshop-api/client.rb]: you can also eliminate the need for changing the url in the tests that way

**Review by hyxnat — CHANGES_REQUESTED** [review]

**Review by hyxnat — APPROVED** [review]

**Pull request merged**